zubhub_frontend/zubhub/package.json

```diff
@@ -16,6 +16,7 @@
     "axios": "^0.27.2",
     "classnames": "^2.2.6",
     "compressorjs": "^1.0.7",
+    "concurrently": "^7.6.0",
```

Comment:
Why have we added this? We want to avoid new dependencies being added unless really required.

Reply:
Oh sorry! that's my mistake. Actually I have added that when I was trying to run both backend and frontend simultaneously on my computer. Now I have fixed it.